This patch is currently disabled. Not stripping binaries until 
debian/rules is now handled through 01_configure.diff 

Since imake+xmkmf rebuilds Makefiles everytime, this is much better,
because otherwise Makefile is changed back by imake+xmkmf during build
and this patch can no longer be reverted.

Index: ivtools-1.2/Makefile
===================================================================
--- ivtools-1.2.orig/Makefile	2009-09-05 22:06:50.000000000 -0400
+++ ivtools-1.2/Makefile	2009-09-05 22:07:06.000000000 -0400
@@ -198,7 +198,7 @@
         TIFFLIBBASE = libTIFF.so
 
             INSTALL = install
-       INSTPGMFLAGS = -s
+       INSTPGMFLAGS = 
        INSTBINFLAGS = -m 0755
        INSTUIDFLAGS = -m 4755
        INSTLIBFLAGS = -m 0644
